src/commons/constants.js

```diff
@@ -2,8 +2,10 @@ export const CELL_TYPE_STRING = 'string';
 export const CELL_TYPE_NUMBER = 'number';
 export const validTypes = [CELL_TYPE_STRING, CELL_TYPE_NUMBER];
 
+export const MISSING_KEY_TITLE = 'Zipcelx config missing property sheet title';
```

Comment:
Minor, place MISSING_KEY_TITLE below MISSING_KEY_FILENAME as you've done the same with INVALID_TYPE_*.